src/Orleans.CodeGeneration/ApplicationPartManagerCodeGenExtensions.cs

```diff
@@ -16,9 +19,11 @@ public static class ApplicationPartManagerCodeGenExtensions
         /// Generates support code for the the provided assembly and adds it to the builder.
         /// </summary>
         /// <param name="manager">The builder.</param>
+        /// <param name="logger">optional logger</param>
         /// <returns>A builder with support parts added.</returns>
-        public static IApplicationPartManagerWithAssemblies WithCodeGeneration(this IApplicationPartManagerWithAssemblies manager)
+        public static IApplicationPartManagerWithAssemblies WithCodeGeneration(this IApplicationPartManagerWithAssemblies manager, ILogger logger = null)
```

Comment:
if user want to enable processing time measure on runtime code gen, they need to provider a logger

Reply:
Ah thanks for adding this, it can be useful. The doc comment should start with a capital.